documentation/source/user_section/tutorials/4.rst

```diff
@@ -0,0 +1,26 @@
+.. _registering-multiple-images:
+
+Tutorial 4: Registering additional images from the same session (e.g. DT, MT data)
```

Comment:
What is DT/MT data?

Reply:
Addressed here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3094/commits/b951cfacf8343be5742d1da68117c717ef6568e8 in `documentation/source/user_section/tutorials/registering-additional-contrasts.rst`

Thanks @joshuacwnewton !